iceoryx_posh/cmake/tiny_json/CMakeLists.txt.in

```diff
@@ -0,0 +1,58 @@
+# Copyright (c) 2020 by Robert Bosch GmbH. All rights reserved.
```

Comment:
Is it correct that some files have Copyright 2021 and other Copyright 2020?

Reply:
Fixed all copyright years to 2021